src/main/java/in/org/projecteka/hiu/consent/ConsentRepository.java

```diff
@@ -36,21 +36,32 @@
             " VALUES ($1, $2, $3, $4, $5)";
     private static final String UPDATE_CONSENT_ARTEFACT_STATUS_QUERY = "UPDATE " +
             "consent_artefact set status=$1, date_modified=$2 where consent_artefact_id=$3";
+    //TODO
     private static final String INSERT_CONSENT_REQUEST_QUERY = "INSERT INTO " +
             "consent_request (consent_request, consent_request_id) VALUES ($1, $2)";
+    //TODO
```

Comment:
TODO?

Reply:
left comments. mostly for future refactoring when we work on the /consents/notify